backend/corpora/dataset_processing/remaster_cxg.py

```diff
@@ -131,103 +227,59 @@ def compute(**kwargs):
-                    logger.info("sparse->sparse")
+            with tiledb.open(f"{cxg}/{target_array}r", "w") as new_X_r:
+                with tiledb.open(f"{cxg}/{target_array}c", "w") as new_X_c:
                     i, chunk = 0, 200_000
                     while i < old_X.shape[0]:
-                        logger.info(f"Chunk {i}")
                         dat = old_X[i : i + chunk]
-                        logger.info("Got dat")
-                        new_X[dat["obs"], dat["var"]] = dat[""]
+                        new_X_r[dat["obs"]] = {"var": dat["var"], "": dat[""]}
+                        new_X_c[dat["var"]] = {"obs": dat["obs"], "": dat[""]}
                         i += chunk
-                elif in_sparse and not out_sparse:
-                    raise ValueError("sparse->dense not supported")
-                elif not in_sparse and out_sparse:
-                    logger.info("dense->sparse")
-                    i, chunk = 0, 20_000
-                    while i < old_X.shape[0]:
-                        logger.info(f"Chunk {i}")
-                        dat = old_X[i : i + chunk]
-                        datnz = np.nonzero(dat)
-                        new_X[datnz[0] + i, datnz[1]] = dat[datnz[0], datnz[1]]
-                        i += chunk
-                else:
-                    logger.info("dense->dense")
-                    new_X[:] = old_X[:]
 
             logger.info("consolidating...")
-            tiledb.consolidate(f"{cxg}/{target_array}")
-            tiledb.vacuum(f"{cxg}/{target_array}")
+            for suffix in ["r", "c"]:
+                tiledb.consolidate(f"{cxg}/{target_array}{suffix}")
+                tiledb.vacuum(f"{cxg}/{target_array}{suffix}")
 
         logger.info(f"Took {time.time() - st} s")
 
     logger.info("done.")
+    return True
 
 
-def choose_X_encoding(cxg, source_array, sparse_threshold, ctx):
-    logger.info("Reading array to determine sparsity...")
-    with tiledb.scope_ctx(ctx):
-        with tiledb.open(f"{cxg}/{source_array}", "r") as X:
-            nnz = 0
-            i, chunk = 0, 200_000
-            while i < X.shape[0]:
-                logger.info(f"Chunk {i}")
-                dat = X.query(dims=[])[i : i + chunk]
-
-                if X.schema.sparse:
-                    nnz += len(dat[""])
-                else:
-                    nnz += np.count_nonzero(dat)
-                i += chunk
-
-            size = X.shape[0] * X.shape[1]
-            logger.info(f"sparsity={1.0 - nnz / size}, nnz={nnz}, extent=({X.shape[0]}, {X.shape[1]})")
-            if 1.0 - nnz / size >= sparse_threshold / 100.0:
-                return "sparse"
-
-            return "dense"
-
-
-def create_new_X(cxg, target_array, kind, compression, X_extent, old_schema, cell_order, tile_order, capacity):
-    is_sparse = kind == "sparse"
-    logger.info(f"create_new_X: {is_sparse}")
+def create_new_X(cxg, target_array, compression, X_extent, old_schema, cell_order, tile_order, capacity):
```

Comment:
suggestion: this didn't originate with your PR, so this is non-blocking but it would be helpful to add docstrings to the functions in this file that are missing them